aws/resource_aws_directory_service_conditional_forwarder_test.go

```diff
@@ -69,21 +69,20 @@ func testAccCheckAwsDirectoryServiceConditionalForwarderDestroy(s *terraform.Sta
 			RemoteDomainNames: []*string{aws.String(domainName)},
 		})
 
+		if isAWSErr(err, directoryservice.ErrCodeEntityDoesNotExistException, "") {
+			continue
+		}
+
 		if err != nil {
-			if isAWSErr(err, directoryservice.ErrCodeEntityDoesNotExistException, "") {
-				return nil
-			}
 			return err
 		}
 
 		if len(res.ConditionalForwarders) > 0 {
 			return fmt.Errorf("Expected AWS Directory Service Conditional Forwarder to be gone, but was still found")
 		}
-
-		return nil
```

Comment:
Previously this `CheckDestroy` function was not properly looping through multiple resources.

Reply:
:heavy_check_mark: